src/main/java/com/example/onlinebookstore/model/User.java

```diff
@@ -0,0 +1,25 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use @Data for entities
```suggestion
@Table(.....
```

Reply:
See comment above